beginner/sum_earnings_sharan-nyn.py

```diff
@@ -0,0 +1,15 @@
+def sum_earnings(input):
+	sum = 0
+	listofnum = input.split(',')
+	for index in listofnum:
+		if (not index.isdigit()):
```

Comment:
this part 
```py
if (not index.isdigit()):
      print('not', index)
      sum = 0
      break
```
causes it to break when it encounters a negative number

`0,1,-2,3` should return `3` but returns `0` because the `-2` gets caught by `isdigit()`